book_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
+require './book'
+
+describe Book do
+  let(:book) { Book.new('Title', 'Author') }
+
+  context 'when initializing a book' do
+    it 'should have the correct title and author' do
+      expect(book.title).to eq('Title')
+      expect(book.author).to eq('Author')
+    end
+
+    it 'should have an empty rentals array' do
+      expect(book.rentals).to be_empty
+    end
+  end
+end
```

Comment:
- [ ] kindly note that you are to test methods in a class too, so please add  a test case for method `add_rental`